### PR TITLE
:lipstick: BDX.io

### PR DESCRIPTION
**Pull request by guillaumecrespel**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
+![Screencapture](https://i.goopics.net/01nxw.png)
 
 ## Sources
  - [reactive programming -medium- @tkssharma](https://medium.com/@tkssharma/reactive-programming-rx-js-introduction-32bf963eee1b)
  - [xstream README.md](https://github.com/staltz/xstream)
  - [cycle.js.org documentation](https://cycle.js.org/getting-started.html)
 
 ## Slides
- - [Web2day 2017](https://docs.google.com/presentation/d/1ffUXgs4dimSTOKi3RtfW_6FYeabH9mQs19S1Om2-73k/edit?usp=sharing)
+ - [BDX.io 2017](https://docs.google.com/presentation/d/1QipKxyUIcntcfenZPb6IzpMv11nKSCUyfVfB5VIgaWI/edit?usp=sharing)
```
Comment (fabienjuif) [inline README.md]: Is it the branch to merge or one living branch ?

Comment (guillaumecrespel) [reply README.md]: one living

Comment (fabienjuif) [reply README.md]: Ok, we should update the master to add slides too then

Comment (fabienjuif) [reply README.md]: And to refert to this branch, nice job 👍

File: README.md [hunk 2]
```diff
@@ -1,13 +1,16 @@
...
 
 ## Sources
  - [reactive programming -medium- @tkssharma](https://medium.com/@tkssharma/reactive-programming-rx-js-introduction-32bf963eee1b)
  - [xstream README.md](https://github.com/staltz/xstream)
  - [cycle.js.org documentation](https://cycle.js.org/getting-started.html)
 
+## Talk
+ - [BDX.io 2017](https://www.youtube.com/watch?v=-tYCelZCKbU&index=20&list=PLUJzERpatfsW5PbUcMAIOIqfCoqS7K1F1)
+
 ## Slides
```
Comment (fabienjuif) [inline README.md:12]: Decks

File: README.md [hunk 3]
```diff
@@ -1,13 +1,16 @@
 # Cycle Music
-![Screencapture](http://zupimages.net/up/17/45/x655.png)
+![Screencapture](https://i.goopics.net/01nxw.png)
 
 ## Sources
  - [reactive programming -medium- @tkssharma](https://medium.com/@tkssharma/reactive-programming-rx-js-introduction-32bf963eee1b)
  - [xstream README.md](https://github.com/staltz/xstream)
  - [cycle.js.org documentation](https://cycle.js.org/getting-started.html)
 
+## Talk
```
Comment (fabienjuif) [inline README.md:9]: Talks

**Pull request closed without merging**